eclass/virtualx.eclass

```diff
@@ -0,0 +1,260 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: virtualx.eclass
```

Comment:
why is this necessary? what issue(s) does it fix?

Reply:
For quite some time now, repoman and CI both spew out the following error for each instance of inheriting virtualx.eclass (interestingly, not for the tree):

`DEPEND: USE flag 'prefix' referenced in conditional '!prefix?' is not in IUSE`

Reply:
this has been fixed in portage/repoman